charts/nfs-subdir-external-provisioner/values.yaml

```diff
@@ -12,6 +12,8 @@ nfs:
   path: /nfs-storage
   mountOptions:
   volumeName: nfs-subdir-external-provisioner-root
+  # Reclaim policy for the main nfs volume
+  reclaimPolicy: Retain
```

Comment:
@olifly - could you please update this new value in the chart [README.md](https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/blob/master/charts/nfs-subdir-external-provisioner/README.md) as well.

Reply:
Ah yes of course! 

Updated